.github/workflows/ci.yml

```diff
@@ -95,6 +95,10 @@ jobs:
           - os: "windows-latest"
             python-version: "pypy-3.8"
     steps:
+      - name: Install a locale for test
+        run: |
+          sudo apt -y update
+          sudo apt install language-pack-de
```

Comment:
that will take a long time and will cause the cache of CI to swell. Is there a different locale that is supported on all platforms without extra installation?

Reply:
No, by default ubuntu ships pretty minimal:

```
chungus:~ ee$ docker run --rm -it ubuntu bash
root@8d6202f9599e:/# locale -a
C
C.utf8
POSIX
```

Not sure what might be available in GitHub actions.

Reply:
1) I'm not certain it will take a "long time"
2) It doesn't look like apt is being cached

Would you mind letting it run to determine if this works, and see what the time penalty is?

Reply:
It's certainly not free,

locally, `time docker run --rm -it ubuntu bash -c "apt update -y && apt install -y language-pack-de"` takes ~13s

Reply:
https://github.com/Backblaze/b2-sdk-python/pull/386/commits/9825eb9dbe7be3afee07258c14deb3381a82c18f should settle both concerns.